Makefile

```
@@ -346,12 +354,12 @@ endif
 # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
 .PHONY: catalog-build
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	$(OPM) index add --container-tool $(CONTAINER_RUNTIME) --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
 
 # Push the catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
-	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+	$(MAKE) $(CONTAINER_RUNTIME)-push IMG=$(CATALOG_IMG)
```

```suggestion
	$(MAKE) docker-push IMG=$(CATALOG_IMG)
```
I believe `docker-push` here is referencing to a make target.  If the `$(CONTAINER_RUNTIME)` value is `podman` then the execution will fail with this change as there will be no `podman-push` make target.